packages/react-composites/src/composites/CallWithChatComposite/PeoplePaneContent.tsx

```diff
@@ -127,22 +267,57 @@ export const PeoplePaneContent = (props: {
         )}
       </Stack>
     );
-  }
-  return (
-    <Stack tokens={peoplePaneContainerTokens}>
-      {inviteLink && (
-        <Stack styles={copyLinkButtonStackStyles}>
-          <DefaultButton
-            text={strings.copyInviteLinkButtonLabel}
-            onRenderIcon={() => <CallWithChatCompositeIcon iconName="Link" style={linkIconStyles} />}
-            onClick={() => copy(inviteLink)}
-            styles={copyLinkButtonStylesThemed}
-          />
+  };
+
+  const desktopViewAddPeopleButton = (inviteLink?: string): JSX.Element => {
+    /* @conditional-compile-remove(PeoplePaneDropdown) */
+    return (
+      <Stack>
+        <DialpadModal isMobile={false} strings={dialpadModalStrings} isModalOpen={isModalOpen} hideModal={hideModal} />
```

Comment:
I have requeseted in ealier comments to extract helper components here. That is also necessary to minimize the conditional compilation directives you need.

Beyond the API (which in this case simply means the new strings and icon name), you might get away with just 1 conditional compilation directive: To add the "add people" component that internally contains the modal and tracks the state for modal being shown or not etc.

Beyond that all helper components, and the Dialpad component itself should be implemented without any conditional compilation (it's just not used for stable builds).